pkgs/top-level/all-packages.nix

```diff
@@ -4682,6 +4682,8 @@ with pkgs;
     openssl = null;
   };
 
+  opencore-amr = callPackage ../development/libraries/libopencore { };
+  
```

Comment:
You should move this to the other libraries in all-packages.nix and sort in it alphabetically. Search for `###` to find the `Development / Libraries` section.

Also you should call it either `libopencore` or `opencore-amr`. So either change the package path or change the attribute name.